File: 2452-words-within-two-edits-of-dictionary/2452-words-within-two-edits-of-dictionary.java
class Solution {
    public List<String> twoEditWords(String[] queries, String[] dictionary) {
        List<String> ans=new ArrayList<>();
        for (String s:queries){
            for(String j:dictionary){
                int diff=0;
                for(int i=0;i<s.length();i++){
                    if(s.charAt(i)!=j.charAt(i)){
                        diff++;
                    }
                }
                if(diff<=2){
                    ans.add(s);
                    break;
                }
            }
        }
        return ans;
        
    }
}

File: 2452-words-within-two-edits-of-dictionary/2452-words-within-two-edits-of-dictionary.py
class Solution:
    def twoEditWords(self, queries: List[str], dictionary: List[str]) -> List[str]:
        ans=[]
        for q in queries:
            for d in dictionary:
                diff=sum(a!=b for a,b in zip(q,d))
                if diff<=2:
                    ans.append(q)
                    break
        return ans
        